contracts/contracts/strategies/balancer/BalancerMetaPoolStrategy.sol

```diff
@@ -214,30 +215,13 @@ contract BalancerMetaPoolStrategy is BaseAuraStrategy {
         );
 
         for (uint256 i = 0; i < _strategyAssets.length; ++i) {
```

Comment:
Fun to think about what would happen if we passed in the same asset multiple times in the _strategyAssets array and _strategyAmounts array.

I don't think there would be an issue. The last one would set the amount, but it would try to transfer each of the amounts out, which should fail, since only the last amount was withdrawn from the strategy.

Reply:
Yes that is a great thought. Quickly glancing through the code I would expect this inner loop: https://github.com/OriginProtocol/origin-dollar/blob/sparrowDom/balancer-sfrxETH-stETH-rETH/contracts/contracts/strategies/balancer/BalancerMetaPoolStrategy.sol#L265-L293 to only consider the last entry in the `strategyAssets` & `_strategyAmount` of the duplicated assets to be considered. And it would withdraw too little from the balancer pool and pay too much in BPT tokens. 

And as you've said the ERC20 transfer out as the final step of the function should fail.